receiver/sqlserverreceiver/logs-documentation.md

```diff
@@ -0,0 +1,16 @@
+# Top-Query Collection Exported Logs Attributes
+
+| Attributes               | Description                                                                                                                                                                                                                                                                                                        | Type   |   |   |   |   |   |   |   |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|---|---|---|---|---|---|---|
+| sqlserver.total_elapsed_time    | Total elapsed time, reported in microseconds (but only accurate to milliseconds), for completed executions of this plan.<br>                                                                                                                                                                                       | int    |   |   |   |   |   |   |   |
+| sqlserver.total_rows            | Total number of rows returned by the query. Can't be null.<br><br>Always 0 when a natively compiled stored procedure queries a memory-optimized table.                                                                                                                                                             | int    |   |   |   |   |   |   |   |
+| sqlserver.total_worker_time     | Total amount of CPU time, reported in microseconds (but only accurate to milliseconds), that was consumed by executions of this plan since it was compiled.<br><br>For natively compiled stored procedures, total_worker_time might not be accurate if many executions take less than 1 millisecond.               | int    |   |   |   |   |   |   |   |
```

Comment:
IIUC, we are reporting milliseconds, not microseconds

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37958/files#diff-82ce9c3c07ed1c4e3d020692468d443c68fbbaaef382fc15a4b6b862f8205fabR466-R475

Reply:
updated